### PR TITLE
Add symfony 5 compatibility

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Add Symfony 5 compatibility

#### Why?

Because Sulu also supports Symfony 5

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,21 +6,21 @@
     "require": {
         "php": "^7.2",
         "sulu/sulu": "^2.0",
-        "ramsey/uuid": "^3.5"
+        "ramsey/uuid": "^3.1"
```
Comment (alexander-schranz) [inline composer.json:9]: @luca-rath can you define here all used Symfony components (range `^4.3 || ^5.0`) atleast here should be `config` and `dependency-injection`

Comment (alexander-schranz) [reply composer.json:9]: Also `http-foundation`, `http-kernel` and `routing` I think is needed

File: .travis.yml [hunk 2]
```diff
@@ -16,11 +16,15 @@ matrix:
     - php: 7.2
       env:
         - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+        - APP_DEBUG=1
+        - APP_ENV=test
     - php: 7.3
```
Comment (alexander-schranz) [inline .travis.yml:21]: Would test on `7.4` and use pcov for code coverage: see changes here: https://github.com/sulu/SuluCommunityBundle/pull/127/files#diff-354f30a63fb0907d4ad57269548329e3

File: .travis.yml [hunk 3]
```diff
@@ -16,11 +16,15 @@ matrix:
...
       env:
         - COMPOSER_FLAGS="--prefer-dist --no-interaction"
         - CODE_COVERAGE="--coverage-clover=coverage.clover"
         - PHPSTAN=true
+        - APP_DEBUG=1
+        - APP_ENV=test
```
Comment (alexander-schranz) [inline .travis.yml:27]: APP_DEBUG should not be needed

APP_ENV can be set on [global](https://github.com/sulu/SuluCommunityBundle/blob/381e6c641c082d9a98848f610532b4be98081f6e/.travis.yml#L10-L12) but should also be force in [phpunit.xml.dist](https://github.com/sulu/SuluCommunityBundle/blob/381e6c641c082d9a98848f610532b4be98081f6e/phpunit.xml.dist#L21-L23)

File: Import/Writer/WriterInterface.php [hunk 4]
```diff
@@ -20,13 +20,11 @@ interface WriterInterface
 {
     /**
      * Write given redirect-route to destination.
-     *
-     * @param RedirectRouteInterface $entity
      */
-    public function write(RedirectRouteInterface $entity);
+    public function write(RedirectRouteInterface $entity): void;
```
Comment (alexander-schranz) [inline Import/Writer/WriterInterface.php:24]: /cc @wachterjohannes should we allow that a Interface implementation can return something? Or should we typehint this with :void? Think we should document all Interface changes in the `UPGRADE.md` file.

File: Manager/RedirectRouteManager.php [hunk 5]
```diff
@@ -102,7 +99,7 @@ public function save(RedirectRouteInterface $redirectRoute)
     /**
      * {@inheritdoc}
      */
-    public function delete(RedirectRouteInterface $redirectRoute)
+    public function delete(RedirectRouteInterface $redirectRoute): void
```
Comment (alexander-schranz) [inline Manager/RedirectRouteManager.php:102]: Interface changes see https://github.com/sulu/SuluRedirectBundle/pull/51/files#r444158958

File: Manager/RedirectRouteManagerInterface.php [hunk 6]
```diff
@@ -32,16 +32,12 @@ public function saveByData($data);
 
     /**
      * Delete given redirect-route.
-     *
-     * @param RedirectRouteInterface $redirectRoute
      */
-    public function delete(RedirectRouteInterface $redirectRoute);
+    public function delete(RedirectRouteInterface $redirectRoute): void;
```
Comment (alexander-schranz) [inline Manager/RedirectRouteManagerInterface.php:36]: Interface changes see https://github.com/sulu/SuluRedirectBundle/pull/51/files#r444158958

File: Model/RedirectRouteRepositoryInterface.php [hunk 7]
```diff
@@ -47,15 +47,11 @@ public function findBySource($source);
 
     /**
      * Persist given entity.
-     *
-     * @param RedirectRouteInterface $entity
      */
-    public function persist(RedirectRouteInterface $entity);
+    public function persist(RedirectRouteInterface $entity): void;
```
Comment (alexander-schranz) [inline Model/RedirectRouteRepositoryInterface.php:51]: Interface changes see https://github.com/sulu/SuluRedirectBundle/pull/51/files#r444158958

File: Model/RedirectRouteRepositoryInterface.php [hunk 8]
```diff
@@ -47,15 +47,11 @@ public function findBySource($source);
...
 
     /**
      * Remove given entity.
-     *
-     * @param RedirectRouteInterface $entity
      */
-    public function remove(RedirectRouteInterface $entity);
+    public function remove(RedirectRouteInterface $entity): void;
```
Comment (alexander-schranz) [inline Model/RedirectRouteRepositoryInterface.php:56]: Interface changes see https://github.com/sulu/SuluRedirectBundle/pull/51/files#r444158958

File: Import/Writer/WriterInterface.php (hunk 4 above)
Comment (wachterjohannes) [reply Import/Writer/WriterInterface.php:24]: i would change the interface and document it in the UPGRADE.md

File: Tests/Unit/Controller/RedirectRouteImportControllerTest.php [hunk 9]
```diff
@@ -40,14 +40,13 @@ class RedirectRouteImportControllerTest extends TestCase
...
     {
-        $importFile = $this->importPath . '/' . $this->fileName;
-
         $request = $this->prophesize(Request::class);
 
         $fileBag = $this->prophesize(FileBag::class);
         $request->reveal()->files = $fileBag->reveal();
 
         $file = new File($fileName = __DIR__ . '/import.csv');
+        $importFile = $file->getPathname();
```
Comment (luca-rath) [inline Tests/Unit/Controller/RedirectRouteImportControllerTest.php:49]: https://travis-ci.org/github/sulu/SuluRedirectBundle/jobs/701256884 This error happened without this change

**Pull request merged**